File: Data Structures & Algorithms/jump-game/submission-0.java
class Solution {
    public boolean canJump(int[] nums) {
    int canReach = 0;
    for(int i = 0;i<nums.length;i++){
        if(canReach>=i)
            canReach = Math.max(canReach, i + nums[i]);
    }
    return canReach>=nums.length - 1;
    }
}

File: Data Structures & Algorithms/buy-and-sell-crypto/submission-0.py
class Solution:
    def maxProfit(self, prices: List[int]) -> int:
        max_profit = 0
        get = prices[0]
        for price in prices:
            if price < get:
                get = price
            else:
                max_profit = max(max_profit, price - get)
        return max_profit
        